rfcs/min-relations.md

```diff
@@ -0,0 +1,270 @@
+# Feature Name: `min-relations`
+
+// FIXME
+// Make sure to talk about these:
+// - Reverse relations.
+// - Or support for add_target_filter.
+
+## Summary
+
+Relations connect entities to each other, storing data in these edges. This RFC covers the minimal version of this feature, allowing for basic entity grouping and simple game logic extensions.
+
+## Motivation
+
+Entities often need to be aware of each other in complex ways: an attack that targets another unit, groups of entities that are controlled en-masse, or complex parent-child hierarchies that need to move together.
```

Comment:
I think the `InFrustum(camera_id)` would be especially understandable here. It's easy to see why it can't be handled by simple tag component here.

Reply:
I worry that multiple camera frustums are bit too obscure of a topic for the average gameplay programmer referring to this RFC. I agree that they're a clear example though!